src/webview/cluster/app/sandboxView.tsx

```diff
@@ -413,8 +416,8 @@ export default function addSandboxView(props): JSX.Element {
                         <Tooltip title='Copy token from DevSandbox console page in browser' placement='bottom'>
                             <Button variant='contained' className='button' href={`${currentState.oauthTokenEndpoint}/request`}>Get token</Button>
                         </Tooltip>
-                        <Tooltip title='Login to DevSandbox OpenShift cluster with token from clipboard' placement='bottom'>
-                            <Button variant='contained' className='buttonSecondary' onClick={handleLoginButton}>Login to DevSandbox</Button>
+                        <Tooltip title={loginSandboxTitle} placement='bottom'>
+                            <span><Button variant='contained' disabled={invalidToken} className='button' onClick={handleLoginButton}>Login to DevSandbox</Button></span>
```

Comment:
I would recommend having the `Login to DevSandbox` as Red in color and active once there is token in the clipboard.

We can add a check that the button is only active, once the clipboard has entry in it. It might be possible that clipboard has some false data, so that will basically throw an error during login.

We also need to enable as SHA check that token copied is sha compatible, as the sandbox cluster token are SHA based.

Reply:
At the moment I activate the button only when the token is able to be used to login in without any error. I left a comment on my current approach. Another way to do it would be to just check the format of the clipboard with a regex, then throw an error in a vscode window if the token is invalid?

Reply:
I think doing both checks would make it more solid proof and error handling. The regex approach sounds good to me.

Reply:
Sounds good! The latest change adds both checks